app/AttributeTypes/DateAttribute.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\AttributeTypes;
+
+class DateAttribute extends AttributeBase
+{
+    protected static string $type = "date";
+    protected static bool $inTable = true;
+    protected static ?string $field = 'dt_val';
+
+    public static function fromImport(string $data) : mixed {
+        return $data;
```

Comment:
Don't we need to check for valid date here?

Reply:
Good point!